### PR TITLE
docs: proxy - add excludes via NO_PROXY

### PR DESCRIPTION
**Pull request by morph027**

### CODE REVIEW

**Review by bblommers — CHANGES_REQUESTED** [review]
Hi @morph027, `NO_PROXY` is language specific as far as I know - is it Golang? Definitely not something supported by all languages.

If you do want to add it to the docs, just reword it to make this explicit: `Some languages may support a way to ... , such as NO_PROXY in Golang, ... `

**Comment by morph027** [comment]
You're right, it's an instruction for libs and tool based on [`curl`](https://curl.se/docs/tutorial.html). I'll add an update.

**Comment by morph027** [comment]
We tested it w/ the PHP AWS SDK, boto3 and awscli (ok, based on boto3).

**Review by bblommers — APPROVED** [review]
That works for me. Thank you for contributing @morph027!
